### DISCUSSION TITLE
Does Podman work on ext4?

### DISCUSSION THREAD

**Discussion by Qbicz** [containers/podman#18702] [General]
Hi! I have a question, given my limited experience with Podman and overlay filesystems.

We are running an Embedded Linux system with ext4 filesystem and Podman running as root, using fuse-overlayfs.

We are looking to optimize performance, and found this information: https://www.redhat.com/sysadmin/podman-rootless-overlay

I'd like to understand if it's possible to switch to ext4 + kernel overlay + Podman, or do we need to use XFS filesystem as the Docker documentation says? https://docs.docker.com/storage/storagedriver/overlayfs-driver/
So the question is, does Podman have the same contraint as Docker, having to use XFS to be able to use kernel overlay instead of userspace fuse-overlayfs?

In general, which filesystems can be used for best performance of Podman on eMMC flash?

**Comment by rhatdan** [comment]
I don't know of any issues with ext4 and rootless overlay.  I would figure this will just work.
You will need to do a podman system reset in rootless mode in order to stop using fuse-overlay.
@giuseppe Please confirm.

**Reply by Qbicz** [reply]
Thank you for this advice, then I'm going to give it a try!
